fix: runtime bug (#252)
ant-design/x: components/useXChat/index.en-US.md

| Property | Description | Type | Default | Version |
| --- | --- | --- | --- | --- |

> | agent | Agent generated by useXAgent, If the onRequest method is used, the agent parameter is required | XAgent | - |  |

Reviewer: ```suggestion
| agent | The proxy generated by `useXAgent` requires the `agent` parameter if the `onRequest` method is used. | XAgent | - |  |
```

> | agent | Agent generated by `useXAgent`, If the `onRequest` method is used, the `agent` parameter is required | XAgent | - |  |

Reviewer: ```suggestion
| agent | `agent` parameter is required when using the `onRequest` method in an agent generated by `useXAgent`. | XAgent | - |  |
```
